pkg/apis/flux/v1alpha1/validation/fluxconfig_test.go

```diff
@@ -337,5 +338,46 @@ var _ = Describe("FluxConfig validation", func() {
 				})
 			})
 		})
+		Describe("additionalSecretResources validation", func() {
+			It("should allow specifying nothing", func() {
+				fluxConfig.AdditionalSecretResources = nil
+				Expect(ValidateAdditionalSecretResources(fluxConfig.AdditionalSecretResources, shoot, rootFldPath)).To(BeEmpty())
```

Comment:
All other tests call the root validation func, i.e., `ValidateFluxConfig`.
I believe this is better and more correct, it could catch mistakes related to how the sub-funcs are called.